_docs/hackathon-preperation-checklist.md

```diff
@@ -0,0 +1,52 @@
+The UWB Hacks is a 13-hour event, which can be a little daunting. Not sure how to prepare? Don't worry, this list has you covered.
```

Comment:
Needs a Jekyll template header here for the file to be added to the navigation menu.

```suggestion
---
layout: default
category: Getting Started
title: Preparing to Hack
order: 3
permalink: /preparing.html
---

The UWB Hacks is a 13-hour event, which can be a little daunting. Not sure how to prepare? Don't worry, this list has you covered.
```